src/objects/Object.h

```diff
@@ -0,0 +1,30 @@
+#ifndef OBJECT_H
+#define OBJECT_H
+
+#include "stdlib.h"
+#include <iostream>
+
+class Level;
+
+class Object {
+private:
+    Level* level;
+
+public:
```

Comment:
```suggestion
private:
```
Я знаю, что по коду много где игнорируется сокрытие, из-за чего нарушается инкапсуляция, но давайте постараемся улучшить код вместе.